library/src/main/java/com/afollestad/vvalidator/form/FormResult.kt

```diff
@@ -37,8 +39,11 @@ class FormResult {
   /** Returns errors that occurred during validation. */
   fun errors(): List<FieldError> = errors
 
+  fun values(): List<FieldValue<*>> = values
```

Comment:
It might be kinda cool to be able to lookup fields by name vs having to iterate over values

Reply:
This function should also have documentation